src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/DocumentManagementService.java

```diff
@@ -46,8 +46,7 @@
 import static uk.gov.hmcts.ecm.common.helpers.DocumentHelper.setDocumentTypeForDocument;
 import static uk.gov.hmcts.ecm.common.helpers.DocumentHelper.setSecondLevelDocumentFromType;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.OUTPUT_FILE_NAME;
-import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.DocumentHelper.createDocumentTypeItemFromTopLevel;
-import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.DocumentHelper.setDocumentNumbers;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.DocumentHelper.*;
```

Comment:
For readability we are not using classes with .*
Instead we import them with their exact path.

Reply:
intellij had auto changed it. corrected the settings and fixed with: https://github.com/hmcts/et-ccd-callbacks/pull/1744/commits/d26589ec7455e828a2522d7ba23c8f7ca37a9343